.circleci/config.yml

```diff
@@ -37,6 +37,22 @@ jobs:
                           make check CLANG_FORMAT=clang-format-8
                           ./upload-stats.sh
                   fi
+      - run:
+          name: Build fuzz targets and deploy to Fuzzit
+          command: |
+                  make tests/fuzz-ppm tests/fuzz-jpg tests/fuzz-header tests/fuzz-text tests/fuzz-dstr CLANG=clang-8 CLANG_FORMAT=clang-format-8
+                  # TODO: CHANGE THIS TO CIRCLE SECRET
+                  export FUZZIT_KEY=055d9d3ba41a36fe7e33fee3bf411b2102703c60e16b1756a7222ab529f16dbf8c687a35313f0b1666d8687e891e5c83
+                  export ORG_ID=sjk2xLZ7DGKxBaTDwfZW
+                  wget -O fuzzit https://bin.fuzzit.dev/fuzzit-1.0
+                  chmod a+x fuzzit
+                  ./fuzzit auth $FUZZIT_KEY $ORG_ID
+                  export FUZZIT_ARGS="--type sanity --revision ${CIRCLE_SHA1} --host stretch-llvm8 --branch ${CIRCLE_BRANCH}"
+                  ./fuzzit create job $FUZZIT_ARGS SVuYwmU8bU4wxDxaPTq1 tests/fuzz-ppm
```

Comment:
I assume we don't want all these keys etc... hard coded like this in the end, or are these project IDs?

Reply:
those are just project ids. I only need to change the key to a proper secret (I Used this key just for testing which I'll rotate soon)